### ISSUE TITLE
Alt text for images

### ISSUE THREAD

**Issue by jennifer-hodgson** [hmrc/design-patterns#146]
# Alt text for images

## Overview
We currently have no content pattern for alt text explaining the nature of content of an image for screen readers or when an image cannot be displayed.

[The need for this pattern arose as a result of the HMRC design system hackday.]

**Comment by stevenaproctor** [comment]
Good article about this https://www.deque.com/blog/great-alt-text-introduction/

**Comment by jennifer-hodgson** [comment]
This is surely a x-gov thing - not sure where it could sit best at present.

**Comment by stevenaproctor** [comment]
Maybe https://design-system.service.gov.uk/styles/images/#alt-text

**Comment by mikeash82** [comment]
Continue discussions on Alt text in the [images](https://github.com/alphagov/govuk-design-system-backlog/issues/70) issue on GOV.UK backlog
